main.go

```diff
@@ -182,7 +182,9 @@ func executeArg(cmd *cobra.Command, arg string, w io.Writer) error {
 
 	// initialize glamour
 	var gs glamour.TermRendererOption
-	if style == "auto" {
+	if len(os.Getenv("GLAMOUR_STYLE")) > 0 {
```

Comment:
I think we can skip this check. If `style` equals `auto` we can always use `WithEnvironmentConfig`, as it will also default to `auto` if no environment variable was set. In other cases we would defer to `WithStylePath`.

Reply:
Works for me. Updated.